centreon/packages/ui/cypress/scripts/cypress-cli.sh

```
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+docker run -it --name cypress -v $PWD/../..:/tmp -w /tmp/packages/ui --name=cypress cypress/included:14.1.0 --component --browser electron $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)